buildstockbatch/base.py

```diff
@@ -620,8 +620,10 @@ def validate_openstudio_version(project_file):
         """
         cfg = get_project_configuration(project_file)
 
-        version_rb = os.path.join(cfg['buildstock_directory'],
-            'resources/hpxml-measures/HPXMLtoOpenStudio/resources/version.rb')
+        if not 'os_version' in cfg:
+            return True
```

Comment:
This can be a problem. BSB will use version from here if it is not provided in config. 
https://github.com/NREL/buildstockbatch/blob/515506f7a2fb1e5abb4677c2bcd27631b09271bf/buildstockbatch/base.py#L42

So, if maybe use something like this:
os_version = cfg.get('os_version', BuildStockBatchBase.DEFAULT_OS_VERSION)